src/ui/components/molecules/table-v2/TableV2.module.scss

```diff
@@ -0,0 +1,112 @@
+@import '../../../../globals/ads-variables.scss';
+@import '../../../../globals/fonts.scss';
+
+.table-v2 {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  font-family: Barlow, sans-serif;
+  padding: 16px;
+  width: 100%;
+
+  &.small {
+    gap: 4px;
+    max-width: 466px;
+    padding: 12px;
+  }
+
+  &.medium {
+    gap: 6px;
+    max-width: 626px;
+    padding: 16px;
+  }
+
+  &.large {
+    gap: 8px;
+    max-width: 832px;
+    padding: 20px;
+  }
+}
+
+.header-container {
+  box-sizing: border-box;
+  display: flex;
+  font-size: inherit;
+  margin-bottom: 8px;
+  width: 100%;
+
+  &.small {
+    height: 40px;
+  }
+
+  &.medium {
+    height: 52px;
+  }
+
+  &.large {
+    height: 60px;
+  }
+}
+
+.info-actions-container {
+  align-items: center;
+  display: flex;
+  gap: 2px;
+  width: 100%;
+
+  @media screen and (width <= 768px) {
```

Comment:
you must use  de response-to instead the media query directly with the sizes config in the global variables